### DISCUSSION TITLE
How to make Bootstrap 5 carousel responsive adaptive no fixed height

### DISCUSSION THREAD

**Discussion by Allmedialab** [twbs/bootstrap#32927] [Q&A]
Hi,

I'm looking for a way to make Bootstrap 5 carousel responsive like this Bootstrap 4 example:
https://codepen.io/JacobLett/details/YaBEPB

Even if the height of the image  is changed the overlay text,  arrows and indicators stay in place. How do I achieve this with the Bootstrap 5 carousel? Don't understand why this is changed in V5 anyway!

Thanks!
David

**Comment by harshpandey002** [comment]
Hello @Allmedialab,

You need to use "media queries" and set a "view height" for every window's width breakpoints to make it responsive like Bootstrap 4.
And I'm not sure about overlay text, arrows etc. if you can give a codepen example, that would be great 😄 

And yeah, they should not have changed that in V5😄

Warm Regards,
Harsh
